docs/architecture/tls13-experimental.md

```diff
@@ -66,3 +66,245 @@ together with their level of testing:
+  signature_algorithms           YES   YES
+  use_srtp                       no    no
+  heartbeat                      no    no
+  apln                           no    YES
+  signed_certificate_timestamp   no    no
+  client_certificate_type        no    no
+  server_certificate_type        no    no
+  padding                        no    no
+  key_share                      YES   YES
+  pre_shared_key                 no    YES
+  psk_key_exchange_modes         no    YES
+  early_data                     no    YES
+  cookie                         no    YES
+  supported_versions             YES   YES
+  certificate_authorities        no    no
+  post_handshake_auth            no    no
+  signature_algorithms_cert      no    no
+
+- Supported groups: depends on the library configuration.
+  Potentially all ECDHE groups:
+  secp256r1, secp384r1, secp521r1(0x0019), x25519, x448.
+
+- Supported signature algorithms: depends on the library configuration.
+  Potentially:
+  ecdsa_secp256r1_sha256, ecdsa_secp384r1_sha384, ecdsa_secp521r1_sha512,
+  rsa_pss_rsae_sha256.
+
+- Supported versions: only TLS 1.3
+
+- Support of Mbed TLS SSL/TLS related (not DTLS) features:
+
+  The TLS 1.3 MVP is compatible with all TLS 1.2 configuration options in the
+  sense that when enabling the TLS 1.3 MVP in the library there is no need to
+  modify the configuration for TLS 1.2. Mbed TLS SSL/TLS related features are
+  not supported or not applicable to the TLS 1.3 MVP:
+
+                                         Supported  Comment
+  MBEDTLS_SSL_ALL_ALERT_MESSAGES           no
+  MBEDTLS_SSL_ASYNC_PRIVATE                no
+  MBEDTLS_SSL_CONTEXT_SERIALIZATION        no
+  MBEDTLS_SSL_DEBUG_ALL                    no
+  MBEDTLS_SSL_ENCRYPT_THEN_MAC             n/a
+  MBEDTLS_SSL_EXTENDED_MASTER_SECRET       n/a
+  MBEDTLS_SSL_KEEP_PEER_CERTIFICATE        no
+  MBEDTLS_SSL_RENEGOTIATION                n/a        Not TLS 1.2 dependent
+  MBEDTLS_SSL_MAX_FRAGMENT_LENGTH          no
+  MBEDTLS_SSL_ALPN                         no
+
+  MBEDTLS_SSL_SESSION_TICKETS              no
+  MBEDTLS_SSL_EXPORT_KEYS                  no         Incomplete support
+  MBEDTLS_SSL_SERVER_NAME_INDICATION       no
+  MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH       no
+
+  MBEDTLS_ECP_RESTARTABLE                  no
+  MBEDTLS_ECDH_VARIANT_EVEREST_ENABLED     no
+
+  MBEDTLS_KEY_EXCHANGE_PSK_ENABLED         n/a        Make sense in TLS 1.3
+  MBEDTLS_KEY_EXCHANGE_DHE_PSK_ENABLED     n/a        context but their current
+  MBEDTLS_KEY_EXCHANGE_ECDHE_PSK_ENABLED   n/a        definition is TLS 1.2 only.
+  MBEDTLS_KEY_EXCHANGE_RSA_PSK_ENABLED     n/a
+  MBEDTLS_KEY_EXCHANGE_RSA_ENABLED         n/a
+  MBEDTLS_KEY_EXCHANGE_DHE_RSA_ENABLED     n/a
+  MBEDTLS_KEY_EXCHANGE_ECDHE_RSA_ENABLED   n/a
+  MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED n/a
+  MBEDTLS_KEY_EXCHANGE_ECDH_ECDSA_ENABLED  n/a
+  MBEDTLS_KEY_EXCHANGE_ECDH_RSA_ENABLED    n/a
+  MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED     n/a
```

Comment:
https://github.com/ARMmbed/mbedtls/pull/4811/commits/71f1ed66c2a6c5177be777871b058a11a650795a , 
TLS1.3 redefined them . above are not valid in TLS1.3

Reply:
Added a comment mentioning this.